hamnosys_dicts.py

```diff
@@ -0,0 +1,238 @@
+DescriptionDict = {
+    "1": "name",
+    "2": "start",
+    "3": "end",
+    "4": "Symmetry operator",
+    "5": "Dom hand - Handshape - base form",
+    "6": "Dom hand - Handhape - Thumb position",
+    "7": "Dom hand - Handshape - Bending",
+    "8": "Dom hand - Handposition - Extended finger direction",
+    "9": "Dom hand - Handposition - Palm orientation",
+    "10": "Dom hand - Handlocation - Frontal plane l/r",
+    "11": "Dom hand - Handlocation - Frontal plane t/b",
+    "12": "Dom hand - Handlocation - Distance from the body",
+}
+
+HandlocationFronalPlaneRL = {
+    "0": "",  # left to the left
+    "1": "",  # left
+    # "2" : "", #center
```

Comment:
why some values in dicts are commented?

Reply:
In this case dict is used more like a legend. In fact it could have been removed at all since it consists only from two signs.
see slide 31 in here to get how it works: [slides](https://www.sign-lang.uni-hamburg.de/dgs-korpus/files/inhalt_pdf/HamNoSys_2018.pdf)